docker-compose.yml

```diff
@@ -384,6 +393,19 @@ services:
         <<: *depends_on-healthy
       web:
         <<: *depends_on-healthy
+  vroom:
+    <<: *restart_policy
+    image: "$VROOM_IMAGE"
+    environment:
+      SENTRY_KAFKA_BROKERS_PROFILING: "kafka:9092"
+      SENTRY_KAFKA_BROKERS_OCCURRENCES: "kafka:9092"
+      SENTRY_BUCKET_PROFILES:
+      SENTRY_SNUBA_HOST: "$SNUBA"
```

Comment:
I don't think this reads from environment variable defined here, as this is hardcoded.

https://github.com/getsentry/self-hosted/blob/8aa571cb758406eccb7aefe13b3b7ebf1a34e5c2/docker-compose.yml#L60

Should we also hardcode the address of Snuba API host?

Unless you want to do `"${SNUBA:-http://snuba-api:3128}"`

Reply:
+1, seeing this in CI

https://github.com/getsentry/self-hosted/actions/runs/5015430579/jobs/9018870128?pr=2154

``` shell
+ docker compose --ansi never run --rm web createuser --force-update --superuser --email test@example.com -***
  time="2023-05-19T21:14:23Z" level=warning msg="The \"SNUBA\" variable is not set. Defaulting to a blank string."
```